config/config.yaml

```diff
@@ -83,3 +83,12 @@ openfga:
 registry:
   host: registry
   port: 5000
+influxdb:
```

Comment:
As mentioned in a previous comment, CE repository won't interact with InfluxDB, at least for now. You can remove the changes in the `config` package.

Reply:
will remove it